### DISCUSSION TITLE
How to publish and deploy refine app to Vultr Ubuntu?

### DISCUSSION THREAD

**Discussion by nguyengiap1467** [refinedev/refine#5049] [General]
I not found manual about publish and deploy refine app without Netlify. 
I need a document guide step by step for deploy refine app on Ubuntu.
Help me please !

**Comment by BatuhanW** [comment]
Hello @nguyengiap1467, refine apps are basically react apps and doesn't have any difference on deployment. Please search for `react deployment`, or `nextjs deployment` depends on the framework you use.
